src/Utility.cc

```diff
@@ -205,6 +205,16 @@ namespace ignition
       return _m.data();
     }
 
+    /////////////////////////////////////////////////
+    std::chrono::system_clock::time_point Convert(const msgs::Time &_time)
+    {
+      auto duration = std::chrono::duration_cast<std::chrono::nanoseconds>(
+        std::chrono::duration<double>(_time.sec() + _time.nsec()/1e9));
```

Comment:
does the following work? It seems like it would have less chance for loss of precision:

~~~
auto duration = std::chrono::seconds(_time.sec()) + std::chrono::nanoseconds(_time.nsec());
~~~

Reply:
we could also use `ignition::math::secNsecToTimePoint` from https://github.com/ignitionrobotics/ign-math/pull/150

if we don't want to wait for that to land, then I would add a TODO